pkg/cmd/roachtest/decommission.go

```diff
@@ -257,34 +273,92 @@ func execCLI(
 	return string(buf), err
 }
 
-func runDecommissionAcceptance(ctx context.Context, t *test, c *cluster) {
+// runDecommissionRecommission tests a bunch of node
```

Comment:
(can't comment on line 271 above, but that's what I'm talking about)

Looking into failures of this test recently, I think I saw that the output of failing decommissioning commands was nowhere to be found. I think this was because of `RunWithBuffer` - output doesn't go to artifacts, so the only place we get it is `buf`, but of course we don't typically print `buf` if we get an `err`.

Could you try to use something like this:

```go
	node := c.Node(runNode)
	l, _ := c.l.ChildLogger(cmdLogFileName(timeutil.Now(), node, args...)
	err := c.RunL(ctx, l, node, args...)
	l.close()
	out, _ := ioutil.ReadAll(l.path)
	return string(out), err
```

The difference is that we get the output in a file 100% of the time, so if a decommission invocation just hangs, we'll see where.